helpers/mime.js

```diff
@@ -19,7 +19,7 @@ function getMimeFromFileExtension(fileExtension) {
 }
 
 const knownMimes = ['text/css', 'text/javascript', 'image/png', 'image/jpeg', 'application/json',
-    'text/plain', 'application/xml'];
+    'text/plain', 'application/xml', 'text/yaml'];
```

Comment:
@adong have you tested this with a yaml file ? is `text/yaml` what is returned by library ?

@pritamstyz4ever we do need custom processing based on file types or file names, which his unique to our problem domain.

Reply:
is this the issue of users not being able to preview or download file types of xml/yaml ??

Reply:
Not just yaml. Any displayable files. Right now we are using file extension to do this. But it’s now fool proof. Say Makefile/Dockerfile they don’t have file extensions.

Reply:
we should do some more refactoring of the code and use mime lookup on content-type, post this bug fix.